tools/internal_ci/linux/grpc_bazel_build_in_docker.sh

```diff
@@ -26,3 +26,7 @@ ${name}')
 cd /var/local/git/grpc
 
 bazel build :all //test/... //examples/...
+# This tests that builds with --define=grpc_no_xds=true work. It can be made a
```

Comment:
Please add a TODO that putting another build here is not ideal (see my lengthy comment to see why), and that we should find a better home for the `-define=grpc_no_xds=true` job.

Reply:
Ack. Todo added